datarockets-style.gemspec

```diff
@@ -32,10 +32,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 1.56.3"
+  spec.add_dependency "rubocop", "~> 1.65.1"
   spec.add_dependency "rubocop-graphql", "~> 1.4"
   spec.add_dependency "rubocop-rails", "~> 2.21"
-  spec.add_dependency "rubocop-rspec", "~> 2.24"
+  spec.add_dependency "rubocop-rspec", "~> 3.0.4"
```

Comment:
Since you're upgrading to major version, make sure you followed upgrade instructions: https://docs.rubocop.org/rubocop-rspec/upgrade_to_version_3.html

Reply:
```yml
inherit_from:
  - config/ruby.yml
  - config/rspec.yml
 ```

And inside `config/rspec.yml`
`require: rubocop-rspec` 
So technically I don't need to change anything